src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Employee/EmployeeType.php

```diff
@@ -109,52 +139,80 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
             ->add('firstname', TextType::class, [
+                'label' => $this->trans('First name', 'Admin.Global'),
                 'constraints' => [
                     $this->getNotBlankConstraint(),
                     $this->getLengthConstraint(FirstName::MAX_LENGTH),
                 ],
             ])
             ->add('lastname', TextType::class, [
+                'label' => $this->trans('Last name', 'Admin.Global'),
                 'constraints' => [
                     $this->getNotBlankConstraint(),
                     $this->getLengthConstraint(LastName::MAX_LENGTH),
                 ],
             ])
-            ->add('email', EmailType::class, [
-                'constraints' => [
-                    $this->getNotBlankConstraint(),
-                    $this->getLengthConstraint(EmployeeEmail::MAX_LENGTH),
-                    new Email([
-                        'message' => $this->trans('This field is invalid', [], 'Admin.Notifications.Error'),
-                    ]),
-                ],
-            ])
             ->add('avatarUrl', FileType::class, [
+                'label' => $this->trans('Avatar', 'Admin.Global'),
                 'required' => false,
                 'attr' => [
                     'accept' => 'gif,jpg,jpeg,jpe,png',
                 ],
             ])
             ->add('has_enabled_gravatar', SwitchType::class, [
                 'required' => false,
+                'label' => $this->trans('Enable gravatar', 'Admin.Global'),
+            ])
+            ->add('email', EmailType::class, [
+                'label' => $this->trans('Email address', 'Admin.Global'),
+                'constraints' => [
+                    $this->getNotBlankConstraint(),
+                    $this->getLengthConstraint(EmployeeEmail::MAX_LENGTH),
+                    new Email([
+                        'message' => $this->trans('This field is invalid', 'Admin.Notifications.Error'),
+                    ]),
+                ],
             ])
         ;
 
         if ($options['is_restricted_access']) {
-            $builder->add('change_password', ChangePasswordType::class);
+            $builder->add('change_password', ChangePasswordType::class, [
+                'label' => $this->trans('Change password...', 'messages'),
```

Comment:
I think we should find a domain for this message. Could you show me a screenshot so I can see where it will appear?

Reply:
![image](https://user-images.githubusercontent.com/47781757/101279688-97871300-37cc-11eb-99a2-70c9bdb4831a.png)
Hey, here it is.

Reply:
Thank you! I was not looking at the good PS version, my bad!